integration_tests/models/marts/metrics.yml

```diff
@@ -1,29 +1,37 @@
 version: 2
 
-metrics:
-  - name: new_customers
-    label: New Customers marked 'paying'
-    model: ref('dim_model_7')
-    description: "The number of paid customers using the product"
 
-    calculation_method: count
-    expression: user_id # superfluous here, but shown as an example
+semantic_models:
+  - name: my_model # A semantic model with the name Transactions
+    model: ref('dim_model_7') # References the dbt model named `fact_transactions`
```

Comment:
> References the dbt model named `fact_transactions`

?
it looks like it references dbt_model_7

Reply:
both just copy paste from docs, will remove